Docs/Setup server on localhost.md

```diff
@@ -0,0 +1,30 @@
+## Working with pgAdmin:
+1. Go to pgAdmin after copying the database code.
+ it can be found here:
+ https://github.com/f1nal3/Juniorgram/blob/master/Utility/SQL/Database_Scheme.sql
+2. Right-click on Databases and then Create / Database ..
+3. Create a table with an arbitrary name.
+4. Next, click on the created base and select the Query Tool
+5. Paste the previously copied code and run F5
+
+
+## Work in the editor:
+1. In ServerInfo.cpp, specify the address of the local host
```

Comment:
I'd separate this from the rest cause this really concerns Client and has nothing to do with the server. Can be transferred to the last part with all the client parts there